source/plugin/Assets/GoogleMobileAds/Platforms/Unity/Countdown.cs

```diff
@@ -51,10 +51,14 @@ public void Start()
     // Update is called once per frame
     public void Update()
     {
+        if (countdownText == null || closeButton == null)
+        {
+            Debug.Log("Invalid Prefab");
```

Comment:
Don't need to log invalid prefab here. We log it in Start(), and we don't need to flood the logs in every update loop